src/content/docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-windows.mdx

```diff
@@ -24,70 +29,98 @@ To run our Kubernetes installation for Windows, follow the steps below to:
 Before you install [New Relic's Kubernetes integration](/docs/integrations/kubernetes-integration/get-started/introduction-kubernetes-integration), review the [compatibility and requirements](/docs/integrations/kubernetes-integration/get-started/kubernetes-integration-compatibility-requirements).
 
 <Callout variant="important">
-  When using containers in Windows, the container host version and the container image version must be the same. Our Kubernetes integration supports Windows versions 1809 and 1909.
+  When using containers in Windows, the container host version and the container image version must be the same. Our Kubernetes integration can run on Windows versions LTSC 2019 (1809), 20H2, and LTSC 2022.
```

Comment:
Are we really supporting it? Should still mark this as Beta/Unsupported? @mangulonr

Reply:
We are not. The docs use a very soft tone for this, calling it a "Limited release", which I've kept as-is. What I've done is subtly replacing "supports" with "can run on", just to avoid giving mixed signals.

I am not sure why the "Limited release" wording was chosen, but if we want to change it, I'd do it on a different PR.